### ISSUE TITLE
Feature Request: Distance to Vessel

### ISSUE THREAD

**Issue by SVSabado** [SignalK/signalk-derived-data#114]
Since the CPA/TCPA calculates the distance to each vessel, can it have the option to write that value?  Three checkboxes so the user can select any combination of CPA, TCPA, and/or range.

**Comment by alnavasa** [comment]
looks like this project is abandoned :(

**Comment by tkurki** [comment]
No, this is an open source project, open for contributions. This is not a machine that fulfills every wish.

**Comment by panaaj** [comment]
Distance to vessel was added in #152.
